src/main/scala/cromwell/util/IntUtil.scala

```diff
@@ -0,0 +1,18 @@
+package cromwell.util
+
+object IntUtil {
+
+  val OrdinalSuffixes = List("th", "st", "nd", "rd", "th", "th", "th", "th", "th", "th")
```

Comment:
Yes, I was waiting for the lenthall PR to be merged and switch to the newest snapshot